CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
-## 0.1.0
```

Comment:
I think this was correct, the only release is 0.1.0 and the new changes should be 0.1.1.

Reply:
Sorry, meant to mark this as a question in the PR. I tried to find the changes between `0.1.0` and `0.1.1` and couldn't.
If you say it is correct, I'll revert this